packages/swirl-components/src/components/swirl-popover-trigger/swirl-popover-trigger.tsx

```diff
@@ -14,8 +14,10 @@ export class SwirlPopoverTrigger {
   @Prop() setAriaAttributes?: boolean = true;
   @Prop() swirlPopover!: string | HTMLSwirlPopoverElement;
   @Prop() triggerOnHover?: boolean = false;
+  @Prop() hoverLingerDuration?: number = 200;
```

Comment:
Issue: A default of 200ms would be a breaking change. Can we use undefined or 0 as the default?

Reply:
Why is a default value here a breaking change?

Reply:
If you're already using triggerOnHover in your app it will introduce a 200ms closing delay when you update the Swirl version.

Reply:
i don't think anyone was using it anyways but i removed the default :D